packages/@lightningjs/ui-components/src/components/Column/Column.test.js

```diff
@@ -351,15 +358,38 @@ describe('Column', () => {
       });
     });
 
-    xdescribe('$columnChanged', () => {
-      it('updates column', () => {
-        const spy = jest.spyOn(column, '_update');
-        column.$columnChanged();
-        expect(spy).toHaveBeenCalled();
+    describe('emitting an $columnChanged signal to all ancestors', () => {
+      it('should emit the signal when the Items cross dimension size has changed', async () => {
+        const itemW = 10;
+        [column, testRenderer] = createColumn(
+          {
+            direction: 'column',
+            autoResizeWidth: true,
+            items: [
+              { ...baseItem, w: itemW },
+              { ...baseItem, w: itemW }
+            ]
+          },
+          {
+            spyOnMethods: ['_updateLayout']
+          }
+        );
+
+        jest.spyOn(column, 'fireAncestors');
+        await column.__updateLayoutSpyPromise;
+
+        expect(column.fireAncestors).not.toHaveBeenCalled();
+
+        column.Items.children[0].w = itemW + 10;
+        await column.__updateLayoutSpyPromise;
+
+        expect(column.fireAncestors).toHaveBeenCalled();
```

Comment:
I think this test can be removed or reworded, it doesn't look like `Column.$columnChanged` is invoked. These tests look to be verifying that when an item's `w` changes, `NavigationManager._updateLayout` will run , update the size/positions of the items, and call `this.fireAncestors('$itemChanged');` because the dimensions of the Column changed to fit the new `w` of the largest item. There's already a test that makes these assertions in NavigationManager.test.js, so I think it's safe to just remove this test.